config/eslint.js

```diff
@@ -35,7 +39,7 @@ export default [
       // https://github.com/gajus/eslint-plugin-jsdoc#readme
       "jsdoc/check-access": ["off"],
       "jsdoc/check-alignment": ["error"],
-      "jsdoc/check-examples": ["off"], // TODO: enable & configure when possible
+      "jsdoc/check-examples": ["off"],
```

Comment:
Re-evaluating this `TODO` as part of this PR.

Reply:
The rule still does not work as of [`jsdoc/check-examples@50.6.1`](https://www.npmjs.com/package/eslint-plugin-jsdoc/v/50.6.1) so I'll leave it disabled.

I won't create an issue to enable this because I could not find a suitable way to track when this can be enabled again.